newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -75,6 +76,18 @@ const getTemplatesGridSizeFromWidth = (width: WidthType) => {
   }
 };
 
+const getProjectLineHeight = (width: WidthType) => {
+  let lineHeight;
+  switch (width) {
```

Comment:
this switch seems quite convoluted.
You could just do `const lineHeight = width === 'small' ? 32 : 36`